gems/pending/postgres_ha_admin/ha_admin.rb

```diff
@@ -0,0 +1,120 @@
+require 'postgres_ha_admin/postgres_ha_admin'
+require 'pg'
+require 'linux_admin'
+
+module PostgresHaAdmin
+  class HaAdmin
```

Comment:
This could probably be named better.
Maybe `FailoverMonitor` or just `Monitor`?